qiskit/quantum_info/operators/mixins/multiply.py

```diff
@@ -35,6 +35,9 @@ class MultiplyMixin(ABC):
     def __rmul__(self, other):
         return self._multiply(other)
 
+    def __mul__(self, other):
+        return self.__rmul__(other)
```

Comment:
Using `__rmul__` looks a bit non-trivial, so how about using `_multiply` directly?

Reply:
Thank you. Changed `__rmul__` to `_multiply`.